### DISCUSSION TITLE
原神

### DISCUSSION THREAD

**Discussion by share121** [share121/inter-knot#8] [其他]
![356074376-e780f26c-8501-4d1e-9bbb-0fbc11f2b43c (Medium)](https://github.com/user-attachments/assets/9b3c5657-5489-436a-9675-8b6eb729a61f)

《原神》是由中国游戏公司米哈游开发的一款开放世界冒险角色扮演游戏（RPG），背景设定在一个名为“提瓦特”的幻想世界中。玩家在游戏中扮演一名被称为“旅行者”的神秘角色，与失散的兄弟姐妹一起探索这个广阔的世界，寻找失踪的亲人，并逐步揭开“原神”的真相。

游戏采用赛璐璐风格的卡通渲染和精致风景绘制，拥有出色的昼夜交替和天气效果，角色建模出色且富有特色。在战斗方面，《原神》的核心机制是元素反应，即两种元素相互影响时产生的超载效果和额外的爆炸伤害。此外，角色培养涉及等级提升、突破、天赋技能升级等多个方面，武器与圣遗物也是重要组成部分。

《原神》自2020年9月28日发行以来，迅速在全球范围内获得了巨大成功。它不仅登顶了包括中美日韩在内的全球多个国家或地区和地区的iOS畅销榜榜首，还在Twitter上成为话题度最高的游戏之一，甚至高过《英雄联盟》和《GTA》等国民级游戏。其成功离不开对不同文化的尊重和展示，尤其是融入了大量中国传统文化元素。

游戏中的世界由七个国家组成：蒙达、黎玉、亚麻座、苏梅鲁、凡尔登、纳坦和斯内兹纳亚，每个国家都有独特的元素女神统治。玩家可以通过Archon任务探索这些国家或地区，完成每个国家的章节后，就会进入下一个国家或地区。随着更新的推进，新的事件、与可玩角色的互动活动、世界任务、boss战、领域和日常任务等内容也会被引入。

《原神》还引入了gacha系统，玩家可以使用primogems购买限时人物旗帜或武器。旗帜分为三种类型：Starter旗帜提供十次免费愿望；Character Event Wish旗帜仅限于特定活动的活动；Standard Wish旗帜则包含所有无限武器和角色。

总体而言，《原神》以其独特的设计内容和可圈可点的元素反应玩法，吸引了大量核心玩家和泛二维人群。它不仅是一款顶级画面和音效的游戏，更是一次对中国文化的深度展示和传播。

#### 《原神》的元素反应机制是如何设计的，以及它如何影响游戏玩法？

《原神》的元素反应机制是游戏战斗系统的核心，通过不同元素之间的相互作用可以触发各种特殊效果，从而影响游戏玩法。这些元素包括水、火、冰、雷、草、风和岩七大属性。

### 元素反应的基本类型

1.  **蒸发（增幅反应）** ：使用火元素或水元素触发时，会按照一定比例提高触发反应时的伤害。
2. **冻结**：使用冰元素与火元素或水元素触发时，可以将敌人冻结在原地，使其无法移动并受到持续伤害。
3.  **超载（剧变反应）** ：使用火元素与水元素或雷元素触发时，会在目标区域产生大量伤害，并有可能引发爆炸。
4. **感电**：使用雷元素与水元素触发时，可以对敌人造成持续的电击伤害。
5. **扩散**：以风元素为核心，与冰、火、雷、水元素相互作用，消耗元素能量。
6. **结晶**：以岩元素为核心，与水元素相互作用，生成岩结晶，对敌人造成冰冻效果。
7. **激化**：使用火元素与雷元素触发时，可以增加攻击力，并使敌人在短时间内处于易燃状态。

### 元素反应的影响

1. **战斗策略**：理解并掌握各种元素反应可以帮助玩家制定更有效的战斗策略。例如，在面对高防御的敌人时，可以选择使用冻结反应来限制敌人的行动能力。
2. **角色搭配**：不同的角色具有不同的元素兼容性，合理搭配角色可以最大化元素反应的效果。例如，火元素角色可以与冰元素角色配合使用蒸发反应来提高输出。
3. **资源管理**：元素反应需要消耗元素能量，因此玩家需要合理管理自己的元素能量，避免在关键时刻没有足够的能量触发所需的反应。
4. **伤害计算**：了解不同元素反应的伤害计算逻辑可以帮助玩家在战斗中做出更精准的判断。例如，蒸发反应虽然本身不造成伤害，但可以通过攻击触发来提高后续攻击的伤害。

#### 米哈游在开发《原神》时采用了哪些中国传统文化元素，这些元素是如何被融入游戏的？

在开发《原神》时，米哈游采用了多种中国传统文化元素，并巧妙地将这些元素融入游戏中，以增强游戏的质感和文化底蕴。以下是一些主要的中国传统文化元素及其在游戏中的具体应用：

1. **地区设计**：
   - 璃月地区是《原神》中一个充满浓厚中国风的虚拟城市。从绝云间、奥藏山到璃月港，每一个场景都充满了中国特色的建筑风格和自然景观。
   - 游戏中的服饰设计也体现了中国传统服饰的特点，如锦衣玉带等。

2. **音乐与艺术**：
   - 璃月地区的背景音乐采用了具有中国特色的民乐，为玩家提供了独特的听觉体验。
   - 游戏还引入了皮影戏风格的新玩法，如“纸映戏”解谜玩法，灵感来源于中国传统皮影戏。

3. **角色设计**：
   - 角色设计方面，《原神》中的许多角色都融入了中国传统文化元素。例如，主角之一刻晴的形象设计参考了中国的传统戏曲元素，其服饰和妆容都充满了中国风的美感。
   - 另外，隐山猊兽和舞狮少年等象征传统祥瑞的元素也在游戏中亮相，进一步增强了游戏的文化氛围。

4. **神话与仙侠元素**：
   - 游戏成功地将中国传统神话和仙侠元素融入其中，创造出了独特的游戏世界和故事情节，吸引了广大玩家的关注和喜爱。

5. **非遗项目**：
   - 米哈游通过推出非遗新项目，如皮影戏风格的新玩法，让传统文化以现代化的方式呈现给玩家，使游戏成为文化传承的重要载体。

#### 《原神》中的gacha系统具体是如何运作的，玩家如何有效利用有限的资源进行战斗和探索？

《原神》中的Gacha系统是一种基于概率的抽卡机制，玩家通过投入游戏内货币或真实货币来抽取新的角色、物品或工具。这种机制源自日本的扭蛋机，玩家在投入硬币后会获得一个包装着未知奖励的“蛋”，从而产生一种悬念和期待感。

### Gacha系统的运作方式

1. **基本概念**：
   - **角色与物品**：玩家可以通过参与Gacha系统来获得新的角色和各种有用的物品。
   - **概率**：每个角色或物品都有不同的出现概率，这些概率通常会在游戏内明确标示出来。

2. **特殊货币**：
   - 玩家可以使用游戏内的特殊货币（如原石）进行抽卡，这些货币可以通过完成任务、活动或购买获得。

3. **成瘾性**：
   - Gacha系统具有很强的成瘾性，因为每次抽卡都可能带来令人兴奋的结果，这使得玩家愿意不断投入资源以追求稀有角色或物品。

### 如何有效利用有限资源

1. **优先级选择**：
   - 玩家应根据当前游戏需求和角色的实用性来决定是否进行抽卡。例如，在缺乏关键角色时，可以优先投入资源进行抽卡以补充队伍实力。

2. **合理规划资源**：
   - 玩家需要合理分配自己的资源，避免过度依赖Gacha系统。可以通过完成日常任务和活动来积累资源，而不是全部用于抽卡。

3. **角色养成与搭配**：
   - 在抽到新角色后，需要投入时间和资源进行角色的养成，包括升级、装备和技能提升等。同时，还需要考虑如何将新角色融入现有的队伍中，以最大化其战斗力。

4. **战斗策略优化**：
   - 了解并掌握战斗系统的基本架构和操作技巧，可以提高战斗效率。例如，选择合适的圣遗物套装和武器，以及合理安排队伍成员的位置和技能释放时机。

### 总结

《原神》的Gacha系统是游戏的核心玩法之一，玩家需要通过合理的资源管理和战斗策略来最大化自己的收益。

#### 自2020年9月28日发行以来，《原神》在全球范围内的销售成绩和玩家基础有何变化？

自2020年9月28日发行以来，《原神》在全球范围内的销售成绩和玩家基础发生了显著变化。

从销售成绩来看，《原神》在推出后迅速取得了巨大的成功。根据Sensor Tower的数据，上线首周，《原神》在App Store和Google Play上的收入达到了6000万美元，并且在全球范围内超越了腾讯的《王者荣耀》、《和平精英》和《PUBG Mobile》，成为全球收入最高的移动游戏。此外，上线后的30天内，《原神》吸金2.45亿美元，位居全球第一。截至2024年初，米哈游旗下的《原神》总收入已经达到了37亿美元，打破了过去纪录，成为史上首年营收最高的游戏IP。

在玩家基础方面，《原神》也取得了令人瞩目的成就。截至2022年，《原神》在全球拥有超过五千多万月活跃玩家，其中中国玩家占1693.7万，日本玩家1165.62万，美国玩家1163.14万，韩国玩家375.52万。这些数据表明，《原神》不仅在中国市场取得了巨大成功，而且在海外市场同样受到了广泛欢迎。

此外，《原神》还登顶了多个畅销榜。例如，在推出新角色“可莉”后，《原神》在10月21日登上了中国的畅销榜榜首，并陆续在另外八个国家的畅销榜登顶，包括美国、加拿大、新加坡和韩国等。这进一步证明了其在全球范围内的受欢迎程度。

总结来说，《原神》自2020年9月28日发行以来，在全球范围内取得了显著的销售成绩和庞大的玩家基础。

#### 《原神》对于其他游戏开发者有哪些启示或影响，特别是在开放世界RPG游戏设计方面？

《原神》作为一款成功的开放世界RPG游戏，对其他游戏开发者提供了多方面的启示和影响，尤其是在开放世界RPG的设计方面。

从地图设计来看，《原神》通过精心设计的高自由度探索环境，为玩家带来了极佳的探索体验。其地图不仅面积广阔，而且在细节上也进行了深入的打磨，例如璃月地区的高耸山脉增加了场景的纵深感和解谜设计的空间。这种高质量的开放世界地图设计，使得玩家可以在游戏中获得丰富的探索乐趣，并且提升了整体的游戏可玩性。

《原神》在元素体系上的创新也是其他游戏开发者可以借鉴的重要方面。游戏中的元素体系不仅构成了角色战斗和环境交互的基础，还增强了玩家的游玩趣味性。这种多样化的元素系统让玩家在进行角色培养和战斗时有更多选择和策略，从而提高了游戏的深度和复杂度。

此外，《原神》采用了“免费游玩+付费内购”的模式，这一策略极大地降低了玩家进入门槛，同时也为游戏带来了持续的收入来源。这种商业模式的成功实践，为其他游戏开发者提供了新的思路，即如何在保持游戏免费的同时，通过合理的内购机制实现盈利。

在游戏内容设计方面，《原神》引入了丰富的解谜元素和机关设计，这些设计不仅增加了游戏的可玩性，还与开放世界的地形紧密结合，呈现出多样化的玩法。这种结合不仅丰富了游戏的内容，也提升了玩家的参与度和沉浸感。

《原神》在UI设计上也值得其他游戏开发者学习。其图标设计美观、色彩统一，功能指向清晰，提高了用户体验。简洁明了的社交界面和便捷的操作方式，使得玩家能够更加专注于游戏本身，而不是被复杂的操作所困扰。

总之，《原神》在开放世界RPG游戏设计方面的成功经验，包括地图设计、元素体系、商业模式以及UI设计等方面，都为其他游戏开发者提供了宝贵的参考和启示。

**Comment by share121** [comment]
写得很好，下次不要再写了

**Reply by 75020-maker** [reply]
合影

**Comment by CocoonedSoul** [comment]
大佬合影

**Comment by warning-Kirov-reporting** [comment]
看不懂，这是新游戏的设定嘛，看这文笔像是强有力的人工智能写的。

**Comment by shilijia-siweikongjian** [comment]
大佬合影！

**Comment by FeiXiao-zm** [comment]
捉！

**Comment by kallen1314** [comment]
感觉这个新游戏不如在空洞探索有意思，毕竟真刀真枪地在里面和以骸战斗，主打一个刺激，只可惜本人进不去。求普通人也能在空洞探索的攻略帖！！！

**Reply by jiulongxishui** [reply]
这边建议去空洞调查协会买套好点的抗侵蚀服再做打算

**Comment by Kaohaaa** [comment]
大佬合影！

**Comment by gaojiaobei** [comment]
致敬网创始人，传奇绳匠“share121”

**Comment by wwwwsn147** [comment]
非常好的帖子，这使我的派蒙旋转

**Comment by Lanxey** [comment]
看不懂，是新游戏吗？（

**Comment by Wu-jian-NGX-zhanghao** [comment]
致敬传奇绳匠share121

**Comment by minyue-yu** [comment]
致敬传奇绳匠share121

**Comment by w1-w2-w3** [comment]
致敬传奇绳匠share121

**Comment by molinFireFly** [comment]
捉！

**Comment by xioagong** [comment]
至暗之时，绝灭之刻

**Comment by Blazing-dust** [comment]
致敬传奇绳匠share121

**Comment by Qiling-XwX** [comment]
听起来好棒！什么时候公测！

**Comment by Elysia-Ego-093** [comment]
你是否在寻找书籍《原神：高等元素论》😂

**Comment by queshen1** [comment]
非常好的原神，使我的派蒙旋转

**Comment by Rope-Worker** [comment]
大佬合影！

**Comment by shuyishaoxiancao** [comment]
这位就是up主吗，太强了

**Comment by qiu-1214** [comment]
致敬绳网创始人share121

**Comment by lamtt26** [comment]
牛逼

**Comment by Qijoten** [comment]
大佬合影！😆

**Comment by hzjwork** [comment]
与大佬合影

**Comment by lllliufeng** [comment]
致敬传奇绳匠share121（怎么又顶上来了

**Comment by zzzuser46** [comment]
下周是我和芙宁娜的婚礼，大家记得参加~
![u=730105532,3422549656 fm=253 fmt=auto app=138 f=JPEG](https://github.com/user-attachments/assets/3b822e6c-eee2-4a71-8295-744c1e4c56e3)

**Comment by aminoc12138** [comment]
启动！！！

**Comment by silverwolf510trillion** [comment]
You're right, however, Genshin Impact is ......

**Comment by adsadk12** [comment]
致敬传奇绳匠share121

**Comment by qingfeng1760** [comment]
是新游戏吗？(/ω＼)

**Comment by Elysia325** [comment]
致敬传奇绳匠share121

**Comment by YukariZZZ** [comment]
大佬合影

**Comment by OoTyleroO** [comment]
合影！

**Comment by 290678** [comment]
ai入侵绳网了？

**Comment by langxmo** [comment]
致敬传奇绳匠share121，然后合影

**Comment by VANA2646** [comment]
合影 感谢科普！

**Comment by sdsd122** [comment]
大佬合影！

**Comment by xinaozi** [comment]
致敬传奇绳匠share121，然后合影

**Comment by 123456wanh** [comment]
合影

**Comment by niegong** [comment]
致敬传奇绳匠share121  合影📸📸

**Comment by cpu644** [comment]
致敬传奇绳匠share121

**Comment by wangliuzhe** [comment]
我勒个114514浏览

**Comment by rinsoraa** [comment]
fairy写的（

**Comment by HeroBrine-c** [comment]
致敬传奇绳匠share121（doge）

**Comment by muhan-972** [comment]
大佬合影！

**Comment by su2569** [comment]
是大佬！

**Comment by xianyujun4** [comment]
是新游戏欸！好耶！

**Comment by tuyuxuan** [comment]
哇，捉到原创作者了。是大佬欸！！！

**Comment by ShallowDream12321** [comment]
合影

**Comment by gdio185** [comment]
合影

**Comment by gdio185** [comment]
致敬绳网创建者，传奇绳匠share11

**Comment by yangyuting0516** [comment]
合影！

**Comment by OURANSORD** [comment]
绳网创始人share121，合影

**Comment by YoKiiiiiiiiiiiii** [comment]
大佬合影

**Comment by Meishangzhao** [comment]
绳斤😡莫名其妙的帖子，取关了

**Comment by su2569** [comment]
......



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年08月12日 11:30 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [share121/inter-knot] 原神 (Discussion #8) |

绳斤😡莫名其妙的帖子，取关了

**Comment by Nina123973** [comment]
很棒游戏，什么时候开测？

**Comment by BaCktOZwero** [comment]
万恶的中译英译中😡

**Comment by Joker-JYH** [comment]
致敬传奇绳匠share121！

**Comment by NorthernLightCN** [comment]
米哈游你真是火了，能让绳网创始人来帮你介绍游戏

**Comment by Tabellarii** [comment]
合影喵

**Comment by rusttymud** [comment]
致敬传奇绳匠share121

**Comment by Will-Dyt** [comment]
致敬传奇绳匠share121

**Comment by ShellShield** [comment]
#转人工

**Comment by RaburadoAmon** [comment]
启动！

**Comment by AetherLude** [comment]
原始人，弃洞！

**Comment by qaqotz** [comment]
嗯呢嗯呢！

**Comment by LEOPard824** [comment]
致敬传奇绳网创始人share121

**Comment by njpengqc** [comment]
你说得对，绝区零，启动！

**Comment by yileinafs** [comment]
捉！

**Comment by wdf-86** [comment]
大佬合影

**Comment by 12344856** [comment]
大佬合影！

**Comment by wuxiafeiguang** [comment]
合影！

**Comment by Rainy-poi** [comment]
大佬合影！

**Comment by yindiea** [comment]
致敬传奇绳匠！

**Comment by RuomoPao** [comment]
致敬传奇绳网创始人share121

**Comment by jewlk** [comment]
令人忍俊不禁

**Comment by czzhangxichen** [comment]
无敌了

**Comment by ShengWangYongHu10086CN** [comment]
期待开服

**Comment by shydrogen** [comment]
刚开服，福利很多，推荐别玩

**Comment by Kamisato100** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/4a102d21856350263e83278ac8c48526.png" alt="吃惊" width="75" >

**Comment by Asteracid** [comment]
我勒个亚契教授啊

**Comment by 114514196** [comment]
()

**Comment by Tuskiy** [comment]
写得很好，下次别写了

**Comment by UEG-LmR** [comment]
合影

**Comment by RivinaK** [comment]
ChatGPT，我们喜欢你~（bushi）

**Comment by OttoApocalypse01** [comment]
是大佬，捉<img src="https://bbs-static.miyoushe.com/static/2024/03/21/fcba628ce15730cf0dbf480ec338f646_3898608417719823301.png" alt="丽都漫步-比利" width="75" >

**Comment by Shuyluo** [comment]
吱

**Comment by LeiMoo** [comment]
这位传奇绳匠能否联合俄罗斯白发女高管收购米忽悠，让尼古拉斯大伟当个前台？
